pyanaconda/modules/storage/kickstart.py

```diff
@@ -152,26 +152,32 @@ def parse(self, args):
 
         if tg.iface:
             if not wait_for_network_devices([tg.iface]):
-                raise KickstartParseError(lineno=self.lineno,
-                        msg=_("Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target is not up.") %
-                             {"nic": tg.iface, "iscsiTarget": tg.target})
+                raise KickstartParseError(
+                    lineno=self.lineno,
+                    msg=_("Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" "
+                          "target is not up.") % {"nic": tg.iface, "iscsiTarget": tg.target}
+                )
```

Comment:
The splitting of strings and chaining with formatting arguments looks rather confusing.

Reply:
How about using format() (keeping hints for translators)- does it look better?

Reply:
That looks great! Thank you!